charts/delphai-deployment/templates/mapping.yaml

```diff
@@ -105,7 +105,7 @@ spec:
     - Origin
     max_age: "86400"
     methods: GET, PUT, POST, DELETE, HEAD, OPTIONS, PATCH
-    origins: "https://docs.{{ $.clusterValues.baseDomain }}"
+    origins: "*"
```

Comment:
Lets whilelist domains here instead of allowing all of them?

Reply:
Ok, this will not be a strraight revert of the recent change.